image/daemon/image.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 
 	"github.com/docker/docker/api/types"
-	"github.com/google/go-containerregistry/pkg/v1"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
needed?

Reply:
It is not needed. go imports added it. Anyhow removed alias now.